package/linux-pam/0005-faillock-remove-user-resolution.patch

```diff
@@ -0,0 +1,35 @@
+--- a/modules/pam_faillock/pam_faillock.c
++++ b/modules/pam_faillock/pam_faillock.c
+@@ -374,28 +374,10 @@
+ 		return PAM_IGNORE;
+ 	}
+ 
+-	if ((pwd=pam_modutil_getpwnam(pamh, user)) == NULL) {
```

Comment:
When doing a patch, we need to minimize our changes. To do that, can you please put the deleted code under `#if 0` and add a comment that the change was done by Siklu and mention the Jira ticket PORTF-1547 in the comment?
Also, can you please explain in the comment why remove this part of the code?

Reply:
fixed